style/style.css

```
@@ -158,3 +164,201 @@ section {
 .right-btn {
   right: 103px;
 }
+.news-list-wrap {
+  width: 100%;
+  display: inline-flex;
+  flex-direction: column;
+  align-items: flex-start;
+  background: var(--surface-surface-default, #fff);
+}
+
+.field-tab {
+  margin: 0;
+  width: 100%;
+  display: flex;
+  height: 8px;
+  padding: 16px 0px;
+  align-items: center;
+
+  border: 1px solid var(--border-border-default, #d2dae0);
+  background: var(--surface-surface-alt, #f5f7f9);
+}
+.text-tab {
+  display: flex;
+  height: 40px;
+  padding: 0px 16px;
+  align-items: center;
+  gap: 8px;
+  cursor: pointer;
+}
+.text-tab .text-category-name {
+  color: var(--text-text-weak);
+}
+
+main .news-list-wrap .progress-tab {
+  position: relative;
+  overflow: hidden;
+  cursor: pointer;
+  width: 166px;
+  height: 40px;
+  flex-shrink: 0;
+  background: var(--surface-surface-brand-alt, #7890e7);
+}
+main .news-list-wrap .progress-tab .text-wrap {
+  display: flex;
+  width: 134px;
+  height: 40px;
+  padding: 0px 16px;
+  align-items: center;
+  gap: 8px;
+  flex-shrink: 0;
+  z-index: 1;
+  position: relative;
+}
+main .news-list-wrap .progress-tab .progress-ratio {
+  position: absolute;
+  top: 0;
+  left: 0;
+  background: var(--surface-surface-brand-default, #4362d0);
+  width: 0;
+  height: 40px;
+  flex-shrink: 0;
+  z-index: 0;
+  transition: width 0.5s;
+}
+main .news-list-wrap .progress-tab .text-category-name {
+  display: flex;
+  flex-direction: column;
+  flex: 1 0 0;
+
+  color: var(--text-white-default, #fff);
+}
+
+main .news-list-wrap .progress-tab .text-category-number .present {
+  color: var(--text-white-default);
+}
+main .news-list-wrap .progress-tab .text-category-number .entire {
+  color: var(--text-white-weak);
+}
+
+.hidden {
+  display: none;
+}
+
+.press-news-wrap {
+  display: flex;
+  padding: 24px;
+  width: 100%;
+  flex-direction: column;
+  align-items: flex-start;
+  gap: 16px;
+  border: 1px solid var(--border-default);
+  box-sizing: border-box;
+}
+
+.press-news-wrap .press-info {
+  display: flex;
+  align-items: center;
+  gap: 16px;
+  align-self: stretch;
+}
+
+.press-news-wrap .press-info .press-icon {
+  display: flex;
+  width: 53px;
+  height: 20px;
+  padding-right: 0px;
+  justify-content: center;
+  align-items: center;
+}
+
+.press-news-wrap .press-info .edit-time {
+  color: var(--text-text-default, #5f6e76);
```

다 변수화 해놓으셨으니까 뒤에부분은 뺴도 될것같아요!

```suggestion
  color: var(--text-text-default);
```